apps/src/maze/maze.js

```diff
@@ -228,12 +228,12 @@ function drawMap () {
   pegmanFadeoutAnimation.setAttribute('begin', 'indefinite');
   pegmanIcon.appendChild(pegmanFadeoutAnimation);
 
-  if (Maze.finish_ && skin.goal) {
+  if (Maze.finish_ && (skin.goalIdle || skin.goal)) {
```

Comment:
Could also just use skin.goal, but set goal on the skin to be the .gif for those skins.